### PR TITLE
Install yum on Fedora 22 and higher

### PR DESCRIPTION
**Pull request by djgalloway**
Signed-off-by: David Galloway <dgallowa@redhat.com>

### CODE REVIEW

File: roles/common/tasks/yum_systems.yml [hunk 1]
```diff
@@ -39,6 +39,15 @@
...
     - timezone
 
+# This is temporary to provide reverse compatibility with certain
+# tasks that call yum specifically.
+# Should be deprecated once we move to ansible v2
+- name: Install yum on Fedora 22 and later
+  dnf:
+    name: yum
+    state: present
+  when: ansible_distribution == 'Fedora' and ansible_distribution_major_version >= "22"
```
Comment (zmc) [inline roles/common/tasks/yum_systems.yml]: Does ``ansible_distribution_major_version >= "22"`` work properly?

Comment (djgalloway) [inline roles/common/tasks/yum_systems.yml]: Ah right.  Needs to be converted to integer.

Comment (djgalloway) [inline roles/common/tasks/yum_systems.yml]: Fixed

**Comment by dmick** [comment]
lgtm

**Pull request merged**